docs/REST API/rest-api.md

```diff
@@ -8970,11 +9022,22 @@ Get a list of all software.
         "name": "FirefoxInsall.pkg",
         "version": "125.6",
```

Comment:
```suggestion
        "version": "125.6",
        "fleet_maintained_app_id": 3,
```

Nullable field.

@noahtalerman we need this for telling FMAs and non-FMAs apart on e.g. software details.

Reply:
@iansltx sounds good! Please feel free to tweak my PR. When it's ready to go, please @ mention Rachael (API design DRI) for a second pair of eyes.